pkg/webhook/validation.go

```diff
@@ -133,6 +134,10 @@ func (v *validator) validateBundle(ctx context.Context, bundle *trustapi.Bundle)
 				unionCount++
 			}
 
+			if source.UseDefaultCAs != nil && *source.UseDefaultCAs {
+				unionCount++
+			}
```

Comment:
We can; I suppose without that constraint a user could request the default CAs be included twice and it would be respected. I can't think why anyone would want that though, so it makes sense to limit and we can always relax later if needed.

Will change!

Reply:
This is now implemented along with a couple of extra validation tests!